scripts/setup/install-conda-debian

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Use this script to install conda package manager.
+set -e
+set -x
+
+# Using instructions from https://docs.conda.io/projects/conda/en/latest/user-guide/install/rpm-debian.html
+
+# Install conda public GPG key to trusted store
+curl https://repo.anaconda.com/pkgs/misc/gpgkeys/anaconda.asc | gpg --dearmor > conda.gpg
+install -o root -g root -m 644 conda.gpg /usr/share/keyrings/conda-archive-keyring.gpg
+
+# Check whether fingerprint is correct (will output an error message otherwise)
+gpg --keyring /usr/share/keyrings/conda-archive-keyring.gpg --no-default-keyring --fingerprint 34161F5BF5EB1D4BFBBB8F0A8AEB4F8B29D82806
+
+# Add conda Debian repo
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/conda-archive-keyring.gpg] https://repo.anaconda.com/pkgs/misc/debrepo/conda stable main" > /etc/apt/sources.list.d/conda.list
+
+# Install!
+apt update
```

Comment:
Tangent: should we use `apt` instead of `apt-get` elsewhere in the code too?

Reply:
No, `apt` doesn’t have a stable scripting interface; we should use `apt-get` everywhere.